pkg/virt-controller/watch/vmi.go

```diff
@@ -2088,9 +2089,9 @@ func (c *VMIController) updateVolumeStatus(vmi *virtv1.VirtualMachineInstance, v
 					Preallocated: kubevirttypes.IsPreallocated(pvc.ObjectMeta.Annotations),
 				}
 				filesystemOverhead, err := c.getFilesystemOverhead(pvc)
-				if errors.Is(err, failedToFindCdi) {
+				if errors.Is(err, failedToFindCdi) || errors.Is(err, multipleCdiInstances) {
 					filesystemOverhead = cdiv1.Percent("0.055")
-					log.Log.V(3).Object(pvc).Infof("Didn't find CDI, continuing normally with filesystem overhead of 5.5%%")
+					log.Log.V(3).Object(pvc).Infof("Failed to find CDI (%s), continuing normally with filesystem overhead of 5.5%%", err.Error())
```

Comment:
You can use `Reason(err)`. It can then look like this:

```
log.Log.V(3).Object(pvc).Reason(err).Infof("Failed to detect CDI, continuing with default filesystem overhead of 5.5%%")
```